### DISCUSSION TITLE
How to access non-compiled stan model for computing effective reproduction ratio

### DISCUSSION THREAD

**Discussion by andrybicio** [epiforecasts/EpiNow2#252] [Q&A] [answered]
I'm developing a project for a course at university, and I am stuck at computing the effective ratio Rt. Is there a way to access stan raw files that are not compiled yet? Because I cannot see any "*.stan" in the repo.
I have already computed both generation and delay distributions, but I cannot figure out how to properly write the stan files.

**Comment by seabbs** [comment, marked as answer]
See and lower level folders: https://github.com/epiforecasts/EpiNow2/tree/master/inst/stan

**Reply by seabbs** [reply]
See the tests for examples of pulling these functions out into R for use: https://github.com/epiforecasts/EpiNow2/blob/master/tests/testthat/test-stan-rt.R

**Reply by andrybicio** [reply]
Thank you, I couldn't find the files. Sorry for bothering

**Reply by seabbs** [reply]
No problem at all. Good luck with your course.
